draft-ietf-httpbis-digest-headers.md

```diff
@@ -77,442 +76,413 @@ normative:
 
 informative:
   RFC2818:
-  RFC7231:
   HTTP11: I-D.ietf-httpbis-messaging
   PATCH: RFC5789
 
 --- abstract
 
-This document defines HTTP fields that support integrity checksums.
-The Digest field can be used for the integrity of HTTP representations.
-The Content-Digest field can be used for the integrity of HTTP message content.
-Want-Digest and Want-Content-Digest can be used to indicate a sender's desire to
-receive integrity fields respectively.
+This document defines HTTP fields that support integrity checksums. The
+Representation-Digest field can be used for the integrity of HTTP
+representations. The Content-Digest field can be used for the integrity of
+HTTP message content. Want-Representation-Digest and Want-Content-Digest can be
```

Comment:
Later you put content integrity first with representation integrity second.  Here, it's the other way around.  Maybe try to make that consistent.  (I'd prefer content-first ordering if it came down to a coin toss.)

Reply:
Ah I know what's happened here. I started on the path of putting content digest in my local copy. However, I soon realized that to keep it consistent I would need to propogate changes through the document, which was very disruptive and not critical to this proposal. I thought I unwound all those changes, I missed this bit, sorry. 

I think the ordering of concepts should be tracked in a seperate ticket, WDYT?